packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -308,7 +308,7 @@ const Updates = decorate([
                       </a>
                     </p>
                   )}
-                  <ActionButton btnStyle='info' handler={effects.update} icon='refresh'>
```

Comment:
Why this change?

Reply:
Ho my bad.
Its was for testing something, but i didnt wanted to commit it.
I will remove